source/math/FloatMatrix.ooc

```diff
@@ -45,6 +45,29 @@ FloatMatrix : cover {
 		result
 	}
 
+	setVertical: func (xOffset, yOffset: Int, vector: FloatPoint3D) {
+		version (safe) {
+			if (xOffset < 0 || xOffset >= this width)
+				raise("Column index out of range in FloatMatrix setVertical")
+		}
+		if (this height - yOffset < 3)
+			raise("Element positions exceed matrix dimensions in FloatMatrix setVertical")
```

Comment:
OK. A general question though, what dictates whether a check should or shouldn't be inside a `version (safe)`? It's a bit unclear to me.